datasets/flwr_datasets/partitioner/distribution_partitioner_test.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Test cases for DistributionPartitioner."""
+
+from collections import Counter
+from typing import Any, Dict, List, Tuple, Union
+
+import numpy as np
+import numpy.typing as npt
+import pytest
+
+from datasets import Dataset
+from flwr_datasets.partitioner.distribution_partitioner import DistributionPartitioner
+
+NDArrayInt = npt.NDArray[np.int_]
+NDArrayFloat = npt.NDArray[np.float_]
```

Comment:
This can be imported from common, just like in the class implementation file.

Reply:
Good catch! Missed this somehow.